webpack.renderer.additions.js

```diff
@@ -1,7 +1,16 @@
 const path = require('path');
+const FilewatcherPlugin = require('filewatcher-webpack-plugin');
 
 const ELECTRON_RENDERER_PROCESS_ROOT = path.resolve(__dirname, 'src/renderer/');
 
+let PROCESS_ARGV = process.env.npm_config_argv;
+if (PROCESS_ARGV) {
+  PROCESS_ARGV = JSON.parse(PROCESS_ARGV);
+}
+
+const isDev = PROCESS_ARGV && PROCESS_ARGV.original &&
```

Comment:
Could `electron-is-dev` be used here?

Reply:
Nope because this isn't inside the Electron runtime, this is for Webpack.

Reply:
Ah duh.